specs/schemes/exact/scheme_exact_algo.md

```diff
@@ -0,0 +1,173 @@
+# Scheme: `exact` on `Algorand`
+
+## Summary
+
+The `exact` scheme on Algorand uses the `Algorand Standard Asset (ASA)`, which are native assets (no contract required) on the Algorand blockchain to authorize a transfer of a specific amount from the payer to the resource server. The approach results in the facilitator having no ability to direct funds anywhere but the address specified by the resource server in `paymentRequirements`.
```

Comment:
In the following text we say:

> A group can have transactions of different types, such as `pay` (payment) and `axfer` (asset transfer).

But here in the summary we just talk about ASA, we should include ALGO as well.